src/test/java/tests/ValidLoginTest.java

```diff
@@ -0,0 +1,19 @@
+package tests;
```

Comment:
new Test class not need, because you do manipulation on Login pages, please add this test case into it